### ISSUE TITLE
Make it more obvious that you can't shoot while on a capture point

### ISSUE THREAD

**Issue by lionleaf** [ninjadev/globalgamejam2016#184]
Change player image to something without a gun? Praying animation?

**Comment by iver56** [comment]
Good idea. Alternatively, Sigve suggested that we play click.mp3 whenever a player tries to shoot but can't.

**Comment by sigvef** [comment]
We already play the click sound, but we should add some visual indication as well.
